cloudformation/config-store.yaml

```diff
@@ -1,10 +1,10 @@
-AWSTemplateFormatVersion: '2010-09-09'
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
As Peter suggested, try to use as few quotes as possible to increase readability. Better use single quotes as they are always literal strings.

Reply:
Nevermind, i see you've just changed the prettier config :)